internal/server/data/schema.sql

```diff
@@ -0,0 +1,328 @@
+-- SQL generated by TestMigrations DO NOT EDIT.
+-- Instead of editing this file, add a migration to ./migrations.go and run:
+--
+--     go test -run TestMigrations ./internal/server/data -update
+--
+
+CREATE TABLE access_keys (
+    id bigint NOT NULL,
+    created_at timestamp with time zone,
```

Comment:
Are these timestamps in the current schema? I thought they were unix time, I could be wrong though.

Reply:
Ya, `gorm.AutoMigrate` produces this column type for `time.Time`. In sqlite it uses `datetime`.